locales-da-DK.xml

```diff
@@ -199,8 +199,8 @@
 
     <!-- LONG ROLE FORMS -->
     <term name="director">
-      <single>director</single>
-      <multiple>directors</multiple>
+      <single>instruktør</single>
```

Comment:
"director" refers to film director in English. That's probably not the right translation, is it?